h/services/list_groups.py

```diff
@@ -51,6 +51,27 @@ def all_groups(self, user=None, authority=None, document_uri=None):
 
         return all_open_groups + private_groups
 
+    def associated_groups(self, authority, user=None):
+        """
+        Return a list of groups relevant to this session/profile (i.e. user).
+
+        Return a list of groups filtered on user and authority that the user
+        belongs to or is associated with (user is a creator/moderator/member).
+
+        If the user is None it returns and empty list.
+        """
+        # Return as quickly as possible. Since the user is unkown there will be no
+        # associated groups so might as well avoid the query.
```

Comment:
I think this comment is redundant since the docstring above explains that this function only returns group associated with the current user.